src/validator.js

```diff
@@ -20,14 +15,11 @@ export default (config) => {
     return false;
   }
 
-  if (!Array.isArray(config.sheet.data)) {
-    console.error(INVALID_TYPE_SHEET);
-    return false;
-  }
-
-  if (!childValidator(config.sheet.data)) {
-    console.error(INVALID_TYPE_SHEET_DATA);
-    return false;
+  for (let i = 0; i < config.sheets.length; i++) {
```

Comment:
ES6 syntax might be a bit prettier: 
config.sheets.map((sheet) => {
  if (!Array.isArray(sheet.data)) { 
...